encodings/fastlanes/src/bitpacking/mod.rs

```diff
@@ -88,6 +89,11 @@ impl BitPackedArray {
             ));
         }
 
+        // TODO(ngates): enforce 128 byte alignment once we have a ScalarBufferBuilder that can
+        //  enforce custom alignments.
+        // let packed = ArrayBuffer::new_with_alignment(packed, FASTLANES_ALIGNMENT);
+        let packed = ArrayBuffer::new_with_alignment(packed, ptype.byte_width());
```

Comment:
`new_with_alignment` (and `new`) are weird names for a thing called `ArrayBuffer`, since I kind of expect them to allocate? a la `Vec::new`